contrib/sssd.spec.in

```diff
@@ -576,6 +576,7 @@ enable authentication with passkey token.
 autoreconf -ivf
 
 %configure \
+    --with-runstatedir=$(_rundir) \
```

Comment:
`./configure --help` doesn't propose with option.
But there is `--runstatedir=DIR` available.